src/billing/SpendReport/SpendReport.tsx

```diff
@@ -9,6 +9,13 @@ import { CostCard } from 'src/billing/SpendReport/CostCard';
 import { CloudPlatform } from 'src/billing-core/models';
 import { Select } from 'src/components/common';
 import { Ajax } from 'src/libs/ajax';
+import {
+  AggregatedCategorySpendData,
+  AggregatedWorkspaceSpendData,
+  CategorySpendData,
+  SpendReport as SpendReportServerResponse,
```

Comment:
I wanted to name the interface in the type definitions to match the name in Swagger (`SpendReport`).